app/src/main/java/org/mozilla/fenix/settings/creditcards/CreditCardEditorFragment.kt

```diff
@@ -50,6 +51,13 @@ class CreditCardEditorFragment : Fragment(R.layout.fragment_credit_card_editor)
         )
     }
 
+    override fun onResume() {
+        activity?.window?.setFlags(
```

Comment:
As on `CreditCardSettingFragment` setting `FLAG_SECURE` here might be too late.
Ideally we should set this even before of `super.onStart` and then remove that flag in `onStop`.